### DISCUSSION TITLE
Support for multi - Kubernetes cluster Cassandra Deployments

### DISCUSSION THREAD

**Discussion by AhmadAlKhaldi86** [k8ssandra/k8ssandra#290] [Q&A]
**Type of question**

 Open question
    Support for multi - Kubernetes cluster Cassandra Deployments. Is it available or in the roadmap. If not any suggessions?

**Comment by jsanda** [comment]
Hi @AhmadAlKhaldi86 

There is no support for multi-k8s yet. It is not listed on https://k8ssandra.io/docs/roadmap/ yet but it is definitely something we are looking to address.

@bradfordcp should we have something about this on the roadmap doc?

**Comment by bradfordcp** [comment]
Yes, I expect a healthy update to the roadmap in the next week.

**Comment by bradfordcp** [comment]
To be clear we are looking to have _some_ of the pieces in place for multi-k8s in the short term, but a turn-key experience is further out.
